### PR TITLE
#824 - Email users on added notes/collaborators/owners

### PR DESCRIPTION
**Pull request by tbl73**

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -22,6 +22,40 @@ def new_owner(user, text)
...
+  def added_work(user, work)
+    @user = user
+    @work = work
+    mail to: @user.email, subject: "New FromThePage Work"
+  end
+
+  def page_edited(user, page)
+    @user = user
+    @page = page
+    mail to: @user.email, subject: "Page edited in FromThePage work"
```
Comment (saracarl) [inline app/mailers/user_mailer.rb]: "A page you worked on in work_name has been edited"

File: app/views/user_mailer/work_collaborator.html.slim [hunk 2]
```diff
@@ -0,0 +1,9 @@
+p Hi #{@user.display_name} --
+
+p We just wanted to let you know that #{@work.owner.display_name} added you as a collaborator on #{@work.title}.
```
Comment (saracarl) [inline app/views/user_mailer/work_collaborator.html.slim]: Let's add the collection name here as well, since the work titles can be a bit obscure.

**Comment by tbl73** [comment]
I made those wording changes.

**Comment by tbl73** [comment]
We decided to merge this PR and PR #947 together, so I'm closing this one.

**Pull request closed without merging**